openshift/dataverse-broker.yaml

```diff
@@ -4,31 +4,73 @@ metadata:
   name: dataverse-broker
 objects:
 
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: dataverse-broker-client
+
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: dataverse-broker
+    annotations:
+      kubernetes.io/service-account.name: dataverse-broker-client
+  type: kubernetes.io/service-account-token
+
 - apiVersion: servicecatalog.k8s.io/v1beta1
   kind: ClusterServiceBroker
   metadata:
     name: dataverse-broker
   spec:
-    url: http://dataverse-broker.bdc.svc.cluster.local
+    url: https://dataverse-broker.${NAMESPACE}.svc.cluster.local
```

Comment:
Using namespace parameter here makes it easier to modify the project name, by editing the value of the parameter.

Reply:
Awesome! Creative! This fixes the issue?

Reply:
I set the value of the NAMESPACE parameter to bdc to match everything else.